spring-cloud-dataflow-build/pom.xml

```diff
@@ -90,6 +90,8 @@
 		<maven-dependency-plugin-for-guides.phase>generate-resources</maven-dependency-plugin-for-guides.phase>
 		<groups />
 		<excludedGroups>slow,docker</excludedGroups>
+		<!--Specific Spring framework javadoc version for attach-javadocs plugin-->
+		<spring-javadoc.version>6.1.3</spring-javadoc.version>
```

Comment:
I think `javadoc-spring.version` might be clearer, plus if we end up needing to add another javadoc version property, the prefix keeps them "close" together. Wdyt?